Library/Formula/trend.rb

```diff
@@ -0,0 +1,37 @@
+class Trend < Formula
+  desc "a general-purpose, efficient trend graph"
+  homepage "http://www.thregr.org/~wavexx/software/trend/"
+  url "http://www.thregr.org/~wavexx/software/trend/releases/trend-1.2.tar.gz"
+  sha256 "40b0f58101801c9373d2b1d990eb327a4582e9a96e4eb2b43faf87346f9dd64f"
+
+  depends_on :x11
+  depends_on 'freeglut'
```

Comment:
So two problems then?

1. the pull request should be against the homebrew-x11 repo
1. the dependency should be 'homebrew/x11/freeglut'